runtime/capsules/runtime/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+# Licensed under the Apache-2.0 license
+
+[package]
+name = "capsules-runtime"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+
+[dependencies]
```

Comment:
I would put the dependencies under `[target.'cfg(target_arch = "riscv32")'.dependencies]` so that the build stays clean when you run `cargo build`. You might have to put a target config attribute above `pub mod mctp` though.

Reply:
done